cmd/txtar-addmod/addmod.go

```diff
@@ -36,6 +36,22 @@ import (
 func usage() {
 	fmt.Fprintf(os.Stderr, "usage: txtar-addmod dir path@version...\n")
 	flag.PrintDefaults()
+
+	fmt.Fprintf(os.Stderr, `
+The txtar-addmod command adds a module as a txtar archive to the a
+testdata module directory as understood by the goproxytest package
+(see https://godoc.org/github.com/rogpeppe/go-internal/goproxytest).
+
+The dir argument names to directory to add the module to. If dir is "-",
+the result will instead be written to the standard output in a form
+suitable for embedding directly into a testscript txtar file, with each
+file prefixed with the ".gomodproxy" directory.
+
+In general, txtar-addmod is intended to be used only for very small
+modules - we do not want to check very large files into testdata/mod.
```

Comment:
I ended up hacking together a Go script that produces skeleton mod definitions. i.e. the initial `module` line followed by the `.mod` and `.info`

Might be worth providing this option via a flag?

Reply:
Probably in another PR. I'd like to know exactly how you use this and what semantics you chose though, so I don't need to think that hard about it :)

Reply:
Yes, agreed. Doing this in another PR makes more sense.